public/data/food/ingredients.json

```diff
@@ -2,179 +2,345 @@
   {
     "id": "egg",
     "name": "Oeuf",
-    "category": "animal_product",
+    "category": "material",
+    "categories": [
+      "animal_product"
+    ],
     "default": "AGRIBALU000000003105162",
     "default_origin": "EuropeAndMaghreb",
     "raw_to_cooked_ratio": 0.974,
     "density": 1,
     "inedible_part": 0.2,
     "transport_cooling": "none",
     "visible": true,
-    "variants": {
-      "organic": {
-        "process": "AGRIBALU000000003105163",
-        "beyondLCA": {
-          "agro-diversity": 50,
-          "agro-ecology": 50,
-          "animal-welfare": 50
-        }
-      },
-      "bleu_blanc_coeur": "Egg Bleu Blanc Coeur outdoor system at farm gate"
+    "complements": {
+      "agro-diversity": 10,
+      "agro-ecology": 10,
+      "animal-welfare": 10
+    }
+  },
+  {
+    "id": "egg-organic",
+    "name": "Oeuf bio",
+    "category": "material",
+    "categories": [
+      "organic",
+      "animal_product"
+    ],
```

Comment:
Please, as discussed over Mattermost, by convention the first category is the main one so the client is able to distinguish how ingredients should be grouped by food categories.

I'm re-attaching the capture to illustrate what we're talking of
![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/0daa136a-ca89-4526-8728-8112938f7dd2)

Reply:
If that's not clear, I need:

```json
   "categories": [
      "animal_product",
      "organic"
    ],
```

instead of:

```json
   "categories": [
      "organic",
      "animal_product"
    ],
```